core/server/pom.xml

```diff
@@ -122,6 +122,12 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-client</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty.toolchain</groupId>
+      <artifactId>jetty-jsp-jdt</artifactId>
```

Comment:
@nate-cloudability I found this dependency is under GPL license. Alluxio is under the Apache 2.0 license, and the [Apache Software Foundation's official guideline](http://www.apache.org/legal/resolved.html#category-b) is not to include LGPL licenses. Could you find an alternative to JDT compiler? Thanks.